### PR TITLE
hello.md english

### PR DESCRIPTION
**Pull request by VielenDanke**
Descritpion client creation in english.

### CODE REVIEW

File: en/hello.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+# The first service
+
+## Proto
+
+Micro does not oblige to use protobuf. But for better experience in communication between microservices (client - server) strongly recommended to use .proto files. It insures developers on a both sides they are using the same generated .proto structures and services.
+
+Usually .proto files located in the directory of the project. If amount of services which are using .proto file are growing up, the best way to share .proto files is using any of storage (github, gitlab etc.).
+
+File .proto usually contains from three part:
+- support - contains imports and package names
```
Comment (vtolstov) [inline en/hello.md:10]: May be header?

File: en/hello.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+- service - contains service names and methods with annotations
+- messages - description for sructures, which are using by service
```
Comment (vtolstov) [inline en/hello.md:12]: structures

**Review by vtolstov — COMMENTED** [review]
Sone suggestions

File: en/hello.md (hunk 1 above)
Comment (VielenDanke) [reply en/hello.md:10]: Instead of support? Yeah looks better

File: en/hello.md (hunk 2 above)
Comment (VielenDanke) [reply en/hello.md:12]: instead of messages?

Comment (VielenDanke) [reply en/hello.md:12]: I see, lexical one

**Review by vtolstov — APPROVED** [review]
LGTM

**Pull request merged**